scanner/src/main/kotlin/Scanner.kt

```diff
@@ -119,12 +119,14 @@ abstract class Scanner(
 
         // Add the projects as packages to scan.
         val consolidatedProjects = consolidateProjectPackagesByVcs(ortResult.getProjects(skipExcluded))
-        val consolidatedReferencePackages = consolidatedProjects.keys.map { it.toCuratedPackage() }
+        val consolidatedReferencePackages = consolidatedProjects.keys.toList()
```

Comment:
I thing it would be clear if you did this commit first. Any you could avoid the `toList()` by making `consolidatedReferencePackages` go second below. So, if this was the first commit, the code would look like:

        // Add the projects as packages to scan.
        val consolidatedProjects = consolidateProjectPackagesByVcs(ortResult.getProjects(skipExcluded))
        val consolidatedReferencePackages = consolidatedProjects.keys

        val packagesToScan = ortResult.getPackages(skipExcluded).map { it.pkg } + consolidatedReferencePackages

Reply:
> I think it would be clearer if you did this commit first.
Maybe, but is it worth the effort? Isn't the diff still quite clear?

> you could avoid the `toList()`
Not that it matters in the end, but I didn't want to change the order in which packages are scanned just to get rid of one `toList()`.

Reply:
> Maybe, but is it worth the effort?

Probably yes, as making that change would have taken only slightly more time than writing this sentence.

> Isn't the diff still quite clear?

It's unnecessary complex as you also need to rewrite the code that you added in the commit before.